File: benches/mutations/variants/go/seed1.t3-subtree-sub.go
package main

func summarizeScores(scores []int, threshold int) []string {
	summaries := make([]string, 0)
	runningTotal := 0
	for _, score := range scores {
		if score < threshold {
			continue
		}
		runningTotal += score / 2
		grade := "poor"
		if score >= 90 {
			grade = "excellent"
		} else if score >= 50 {
			grade = "adequate"
		}
		summaries = append(summaries, grade)
	}
	if runningTotal > 250 {
		summaries = append(summaries, "aggregate")
	}
	return summaries
}

File: benches/mutations/variants/typescript/seed1.t3-subtree-sub.ts
function summarizeScores(scores: number[], threshold: number): string[] {
  const summaries: string[] = [];
  let runningTotal = 0;
  for (const score of scores) {
    if (score < threshold) {
      continue;
    }
    runningTotal += score / 2;
    let grade = "poor";
    if (score >= 90) {
      grade = "excellent";
    } else if (score >= 50) {
      grade = "adequate";
    }
    summaries.push(grade);
  }
  if (runningTotal > 250) {
    summaries.push("aggregate");
  }
  return summaries;
}
